livingatlas/pipelines/pom.xml

```diff
@@ -0,0 +1,890 @@
+    <snakeyaml.version>1.25</snakeyaml.version>
+    <streamex.version>0.7.2</streamex.version>
+    <!-- This is overriding GBIF mother pom which is using a 2.9 version which causes issues -->
+    <jackson-module-scala_2.11.version>2.11.0</jackson-module-scala_2.11.version>
+    <jcommander.version>1.78</jcommander.version>
+    <layers-store.version>2.0.3</layers-store.version>
+
+    <!-- Plugins -->
+    <io.fabric8.version>0.33.0</io.fabric8.version>
+    <org.jetbrains.annotations>13.0</org.jetbrains.annotations>
+
+  </properties>
+
+  <scm>
+    <connection>scm:git:git@github.com:AtlasOfLivingAustralia/la-pipelines.git</connection>
+    <url>https://github.com/AtlasOfLivingAustralia/la-pipelines</url>
+    <developerConnection>scm:git:git@github.com:AtlasOfLivingAustralia/la-pipelines.git</developerConnection>
+    <tag>HEAD</tag>
+  </scm>
+
+  <dependencies>
+
+    <dependency>
+      <groupId>org.jetbrains.kotlin</groupId>
+      <artifactId>kotlin-stdlib</artifactId>
+      <version>${okio-kotlin.version}</version>
+      <scope>runtime</scope>
+    </dependency>
+    <dependency>
+      <groupId>org.jetbrains.kotlin</groupId>
+      <artifactId>kotlin-stdlib-common</artifactId>
+      <version>${okio-kotlin.version}</version>
+      <scope>runtime</scope>
+    </dependency>
+    <dependency>
+      <groupId>org.jetbrains</groupId>
+      <artifactId>annotations</artifactId>
+      <version>${org.jetbrains.annotations}</version>
+      <exclusions>
+        <exclusion>
+          <groupId>org.jetbrains.kotlin</groupId>
+          <artifactId>kotlin-stdlib</artifactId>
+        </exclusion>
+      </exclusions>
+    </dependency>
+    <dependency>
+      <groupId>org.apache.beam</groupId>
+      <artifactId>beam-sdks-java-io-solr</artifactId>
+      <version>${apache.beam.version}</version>
+    </dependency>
+    <dependency>
+      <groupId>org.apache.beam</groupId>
+      <artifactId>beam-sdks-java-extensions-join-library</artifactId>
+      <version>${apache.beam.version}</version>
+    </dependency>
+    <dependency>
+      <groupId>commons-configuration</groupId>
+      <artifactId>commons-configuration</artifactId>
+      <version>${commons-configuration.version}</version>
+    </dependency>
+    <dependency>
+      <groupId>io.netty</groupId>
+      <artifactId>netty</artifactId>
+      <version>${netty.version}</version>
+    </dependency>
+    <dependency>
+      <groupId>io.netty</groupId>
+      <artifactId>netty-all</artifactId>
+      <version>${netty-all.version}</version>
+    </dependency>
+
+    <!-- Tools -->
+    <dependency>
+      <groupId>org.projectlombok</groupId>
+      <artifactId>lombok</artifactId>
+      <version>${lombok.version}</version>
+      <scope>provided</scope>
+    </dependency>
+
+    <!-- This project -->
+    <dependency>
+      <groupId>org.gbif.pipelines</groupId>
+      <artifactId>models</artifactId>
+      <version>${gbif-pipelines.version}</version>
```

Comment:
Versions can be removed here, as they're defined in parent poms

Reply:
The version specified in GBIF parent POMs is the ${project.version}. The module version here is 1.0-SNAPSHOT.

Reply:
switched to 2.5.4-SNAPSHOT and removed versions